### DISCUSSION TITLE
When is transmission going to intergrate IPFS compatibility?

### DISCUSSION THREAD

**Discussion by dave90dave** [transmission/transmission#2525] [Ideas]
Hi people,

I thought, that it might be great if transmission will be the first torrent client, that is able to download and upload files/directories on the IPFS network.

How do you all think about adding IPFS?

Do you think that it should be integrated into transmission?

Regards
Dave

**Comment by ile6695** [comment]
[IPFS](https://github.com/ipfs/ipfs) is currently not production-ready.

>How do you all think about adding IPFS?

Transmission is a BitTorrent client. Becoming more than that is unlikely.

**Comment by josephj11** [comment]
Although this isn't exactly the same issue, I wonder if it would be possible to use something like IPFS for communication with other peers. This is handled now via port forwarding, but many/most VPNs do not support port forwarding and torrent performance is drastically reduced without it.
